wiki/Guides/Livestreaming_osu!/en.md

```diff
@@ -0,0 +1,65 @@
+# Livestreaming osu!
+
+This guide will show you how to livestream osu! on Twitch using [OBS Studio](https://obsproject.com/ "Open Broadcaster Software"). Doing so is relatively simple, although it might require a reliable computer able to both run and livestream osu!.
+
+## Twitch
+
+If you have not already, sign up for a [Twitch account](https://www.twitch.tv/signup "Twitch").
+
+Now you will need to setup your account for livestreaming osu!. From the main page, click on your avatar in the top right and select `Creator Dashboard` from the list.
+
+![](img/dashboard.png "Entering the Creator Dashboard")
+
+Creator Dashboard is the place where you can configure your stream information, chat moderation, profile info, and more. For now, we will focus on stream information. From the main page of the dashboard, click on the `Stream Manager` button on the left, and then choose the `Edit Stream Info` option on the right.
+
+A dialog will open for you to enter your stream's information. You can fill the fields however you want; however, the title should comprehend what the stream will be about and also be inviting for people viewing the list of available streams. The category must be set to the game you are playing at the moment, which would be `osu!`.
+
+![](img/stream-info.png "Editing your stream's information")
+
+---
+
+After editing your stream's information, click the `Done` button. Now click on the `Preferences` button on the left, then `Channel`. Look for the `Primary Stream key` field and click `Copy`. **Do not share this key**. For now, paste the copied stream key in a notepad.
```

Comment:
a explanation even so short is always better
```suggestion
After editing your stream's information, click the `Done` button. Now click on the `Preferences` button on the left, then `Channel`. Look for the `Primary Stream key` field and click `Copy`. **Do not share this key — it allows for streaming on your behalf**. For now, paste the copied stream key in a notepad.
```

Reply:
I think the spaces between the emdash should be removed. So it should be like this instead.
```suggestion
After editing your stream's information, click the `Done` button. Now click on the `Preferences` button on the left, then `Channel`. Look for the `Primary Stream key` field and click `Copy`. **Do not share this key—it allows for streaming on your behalf**. For now, paste the copied stream key in a notepad.
```

Reply:
yeah, English punctuation gets me sometimes.